.github/workflows/repostats.yaml

```diff
@@ -20,4 +20,4 @@ jobs:
         uses: jgehrcke/github-repo-stats@dab4a915b37a7521cd54033a3147daaeb868ec5c
         with:
           repository: ${{ matrix.statsRepo }}
-          ghtoken: ${{ secrets.github_token }}
+          ghtoken: ${{ secrets.GHRS_GITHUB_TOKEN }}
```

Comment:
What does `GHRS` mean?

Reply:
Git hub repo stats

Reply:
which i see now makes the github part of github token a little weird

Reply:
Nit: In this case, no need to save keystrokes. We can just list out exactly what it is: `REPO_STATS_TOKEN` is pretty explicit. It will save the next person who comes along the headache of figuring out what it means.